--- origsrc/libetpan-0.54/configure.ac	2008-02-20 16:40:31.000000000 -0600
+++ src/libetpan-0.54/configure.ac	2008-06-22 16:35:19.921875000 -0500
@@ -106,6 +106,7 @@
 
 # Check the C compiler.
 AC_PROG_CC
+AC_PROG_CXX
 
 # Compiler flags.
 AC_ARG_ENABLE(debug,   [  --enable-debug          setup flags (gcc) for debugging (default=no)],
@@ -376,7 +377,7 @@
   case "$DBVERS" in
     3)
     AC_MSG_RESULT([version 3.x or above])
-    for lib in db-4.5 db-4.4 db-4.3 db-4.2 db-4.1 db-4.0 db-4 db4 db-3.2 db-3 db3 db; do
+    for lib in db-4.6 db-4.5 db-4.4 db-4.3 db-4.2 db-4.1 db-4.0 db-4 db4 db-3.2 db-3 db3 db; do
       if test "x$DBLINKED" = "x0"; then
 dnl        AC_CHECK_LIB($lib, db_create, [DBLIB="-l$lib"; DBLINKED=1], [])
 dnl installations of libdb4 function names are defined in db.h
--- origsrc/libetpan-0.54/rules.mk	2008-01-01 08:54:48.000000000 -0600
+++ src/libetpan-0.54/rules.mk	2008-06-22 16:46:06.515625000 -0500
@@ -135,8 +135,8 @@
 	           echo "$(LN_S) -f ../../$(subdir)/$${hdr}"; \
 	           $(LN_S) -f ../../$(subdir)/$${hdr}; \
              else \
-	           echo "$(LN_S) -f ../../$(subdir)/$(srcdir)/$${hdr}"; \
-	           $(LN_S) -f ../../$(subdir)/$(srcdir)/$${hdr}; \
+	           echo "$(LN_S) -f $(srcdir)/$${hdr}"; \
+	           $(LN_S) -f $(srcdir)/$${hdr}; \
              fi; \
            fi; \
 	     done; \
--- origsrc/libetpan-0.54/src/Makefile.am	2007-10-29 19:40:39.000000000 -0500
+++ src/libetpan-0.54/src/Makefile.am	2008-06-22 16:53:28.156250000 -0500
@@ -51,7 +51,7 @@
 
 libetpan_res =
 libetpan_res_ldflag =
-no_undefined =
+no_undefined = -no-undefined
 
 libetpan_deps =
 endif
